VRDR/IJEMortality.cs

```diff
@@ -655,18 +655,18 @@ private void Dictionary_Geo_Set(string ijeFieldName, string fhirFieldName, strin
                     // v1.3 removed lookups for city
                     if (geoType == "place") // This is a tricky case, we need to know about county and state!
                     {
-                        string state = null;
-                        string county = null;
-                        dictionary.TryGetValue(keyPrefix + "State", out state);
-                        dictionary.TryGetValue(keyPrefix + "County", out county);
-                        if (!String.IsNullOrWhiteSpace(state) && !String.IsNullOrWhiteSpace(county))
-                        {
-                            string city = dataLookup.StateNameAndCountyNameAndPlaceCodeToPlaceName(state, county, value);
-                            if (!String.IsNullOrWhiteSpace(city))
-                            {
-                                dictionary[key] = city;
-                            }
-                        }
+                        // string state = null;
```

Comment:
Why is some code deleted and some other code just commented out? I propose that if we comment out some code that we want to leave around in a commented out state that there should always be some comment prose that describes 1) why the code is no longer needed and 2) why we might need it in the future such that commenting it out is the right decision.

Reply:
Code should be deleted...  Deleted it.